src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -11,30 +11,24 @@
 import org.junit.jupiter.api.Test;
 
 public class RegistrationServiceTest {
-    private static final String OK_PASSWORD = "123456";
-    private static final String LOGIN = "Dick";
-    private static final int AGE_OK = 25;
-    private static final int TOO_YOUNG = 16;
-    private static final String SHORT_PASSWORD_NOT_OK = "1234";
-    private static final int NEGATIVE_AGE = -100;
-    private static final int ADULT_AGE_MIN_OK = 18;
     private User firstUser;
     private User secondUser;
-    private RegistrationService service = new RegistrationServiceImpl();
+    private RegistrationService service;
 
     @BeforeEach
     void setUp() {
+        service = new RegistrationServiceImpl();
```

Comment:
what is the difference between `@BeforeAll` and `@BeforeEach` annotations?
![image](https://user-images.githubusercontent.com/36365171/185929870-52fb557e-ac6f-4af2-b461-f7826b293a6f.png)